src/exceptions.py

```diff
@@ -0,0 +1,17 @@
+def FunTypeError(num1, num2, flag):
+    try:
+        int(num1)
+        int(num2)
+    except ValueError:
+        flag = 1
+        print("Введите число")
+    return flag
+
+
+def FunZeroDivisionError(num1, num2, flag):
```

Comment:
переменная flag никак не используется внутри функции, пожалуйста убери ее из аргументов 
```suggestion
def log_zero_division_error(num1, num2):
```